packages/grafana-ui/src/components/ThresholdsEditor/ThresholdsEditor.tsx

```diff
@@ -1,14 +1,22 @@
 import React, { PureComponent, ChangeEvent } from 'react';
-import { Threshold } from '../../types';
+
 import { ColorPicker } from '..';
 import { PanelOptionsGroup } from '..';
+
 import { colors } from '../../utils';
-import { ThemeContext } from '../../themes/ThemeContext';
-import { getColorFromHexRgbOrName } from '../../utils/namedColorsPalette';
+import { getColorFromHexRgbOrName } from '../../utils';
+
+import { ThemeContext } from '../../themes';
+
+import { Threshold } from '../../types';
 
 export interface Props {
   thresholds: Threshold[];
   onChange: (thresholds: Threshold[]) => void;
+  /**
+   * @enableText enable ability to add text to a threshold.
```

Comment:
@peterholmberg I'm 99% sure this won't work, as `enableText` is not a valid jsdoc keyword. Use [`@description`](http://usejsdoc.org/tags-description.html) instead.

Reply:
ah, misunderstood the documentation